eng/pipelines/tools-repo-versioning.yml

```diff
@@ -0,0 +1,24 @@
+# Tag the tools repos (azure-sdk-tools or azure-sdk-build-tools) to create some form of versioning.
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+  - job: TagToolsRepo
+    displayName: Version Repo with Git Tag
+
+    steps:
+    - task: PowerShell@2
+      displayName: 'Verify and Create Tag on Repo'
+      inputs:
+        targetType: filePath
+        filePath: scripts/powershell/version-repo-with-tag.ps1
+        arguments: >
+          -RepoName $(ToolsRepoName) -RepoOrg $(ToolsRepoOrg) -RepoType $(ToolsRepoType)
```

Comment:
Where do these parameters get set? Do we plan to run this same script for both azure-sdk-tools and azure-sdk-build-tools?

Reply:
The variables are set in the pipeline. Yes I am using the same script and using the RepoType variable to condition the logic flow.

Reply:
We should see about getting the info from the default predefined Azure DevOps pipeline variables. I think most of these already exist.